server/app/controllers/api/v1/wishes_controller.rb

```diff
@@ -1,5 +1,5 @@
 require 'uri'
-require 'pry'
+#require 'pry'
```

Comment:
fmi, how does pry help here?

also, minor: I'd recommend removing the line instead of only commenting it out - it can be easily added later if needed. 
If it has to be added and being commented out is only temporary, I usually:
1. create an issue in github to enable back
2. keep the comment and add a reference to the issue in it e.g. "#require 'pry' -- issue #000 will add this back"
That way there is always a reminder both in code and in the issue tracking and makes the chance of addressing it later, when working in a team, higher